frontend/packages/client/src/components/CustomTab.tsx

```diff
@@ -0,0 +1,34 @@
+import { forwardRef } from 'react';
+import { Svg } from '@cast/shared-components';
+import {
+  Flex,
+  HStack,
+  Tab,
+  useMultiStyleConfig,
+  useTab,
+} from '@chakra-ui/react';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const CustomTab = forwardRef<HTMLElement, any>((props, ref) => {
```

Comment:
Should we add an interface above to define the props for the component instead of using any?